src/content/docs/guides/upgrade-migrate/from-tauri-1.mdx

```diff
@@ -1057,3 +1057,39 @@ tauri::Builder::default()
         Ok(())
   })
 ```
+
+## Prepare for Mobile
+
+The mobile interface of Tauri requires your project to output a shared library so if you are targetting mobile for your existing application, you must change your crate to
+produce that kind of artifact along with the desktop executable.
+
+- First, you need to change your Cargo manifest to produce the library. Append the following block:
+
+```toml title=src-tauri/Cargo.toml
+[lib]
+name = "app_lib"
+crate-type = ["staticlib", "cdylib", "rlib"]
+```
```

Comment:
It's a bit weird, but for non-consoles you use this format it looks like
~~~suggestion
```toml
// src-tauri/Cargo.toml
[lib]
name = "app_lib"
crate-type = ["staticlib", "cdylib", "rlib"]
```
~~~